contracts/src/RiscZeroVerifierEmergencyStop.sol

```diff
@@ -23,7 +23,7 @@ import {IRiscZeroVerifier, Receipt} from "./IRiscZeroVerifier.sol";
 
 /// @notice Wrapper for an IRiscZeroVerifier contract, providing emergency stop function.
 contract RiscZeroVerifierEmergencyStop is IRiscZeroVerifier, Ownable2Step, Pausable {
-    IRiscZeroVerifier immutable verifier;
+    IRiscZeroVerifier public immutable verifier;
```

Comment:
I believe this must be `public` if we want to easily inspect this value from Etherscan

Reply:
Ok, and this seems good to me :thum